scripts/jtsync/jtsync.rb

```diff
@@ -153,7 +154,10 @@ def update_card_label(card, job)
 
   return if card.labels.include? label
 
-  card.labels.each { |l| card.remove_label(l) }
+  card.labels.each do |l|
+    next unless STATUS_MAPPING.values.include? l.name
+    card.remove_label(l) 
```

Comment:
There's no point removing `l` here if `l == label`, since then you would immediately re-add it below.